guest-agents/xen-guest-agent/pkg.yaml

```diff
@@ -0,0 +1,37 @@
+name: xen-guest-agent
+variant: alpine
+shell: /bin/sh
+install:
+  - cargo
```

Comment:
okay, i guess this is better than us maintaining rust in our toolchain (until we have a strong reason to do so, since it seemed to fail freqeuntly)

Reply:
Yeah, I couldn't build it using base. This seems better.